invenio_rdm_records/records/jsonschemas/records/record-v2.0.0.json

```diff
@@ -123,7 +123,8 @@
       "additionalProperties": false,
       "required": ["identifier", "provider"],
```

Comment:
shall we move these restrictions to the marshmallow schema? that way we can properly save drafts that have empty values in some fields but make sure requirements are met upon publishing action.